### DISCUSSION TITLE
Transmog skins for quest rewards received before transmog implementation

### DISCUSSION THREAD

**Discussion by Vulturepeak2** [chromiecraft/chromiecraft#3349] [Suggestions]
It would be nice if we don't have to roll a new character to get low level quest reward skins. Especially for someone like me who already has 7 level 60s.

**Comment by Cooties-cc** [comment]
Somewhat agree if it could be worked on to query a players quest completed log, and apply the rewards to what they have done. I didn't think we got all the rewards as mogs, until I started to level again on an alt. I would really appreciate not having to do pic related all over again. 
<a href="https://imgur.com/HQeW2N1"><img src="https://i.imgur.com/HQeW2N1.png" title="source: imgur.com" /></a>

**Comment by Nyeriah** [comment]
Accepted (long ago)
